packages/tools/devtools/devtools-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -95,17 +103,25 @@ export function AudienceHistoryTable(props: AudienceHistoryTableProps): React.Re
 							<LabelCellLayout
 								icon={
 									item.changeKind === "joined" ? (
-										<ArrowJoinRegular />
+										<ArrowJoinRegular
+											style={{ color: setThemeStyle(themeInfo.name) }}
+										/>
 									) : (
```

Comment:
Let's follow the same pattern we're moving to everywhere else to apply styles. You can see an example [here](https://github.com/alexvy86/FluidFramework/blob/4cf880375dec0a3c9ec892a063fa26189bbb6d17/packages/tools/devtools/devtools-view/src/DevtoolsView.tsx#L332-L342).

For this case I'd say instead of `root` we can create a style named `overrideForegroundColorHC` and apply that to the elements that need it. I'd also check if we can apply it on the `<Table>` so we don't have to do it individually to each cell.

Reply:
@alexvy86 I'm not sure it's possible to put conditional logic into a `makeStyles`. I think it'd improve consistency if all styles were in a `makeStyles` as opposed to a function here and there that takes in a prop. But the logic on applying a style only for a specific theme should be inline.

Reply:
@WayneFerrao 

As per our conversation, I also agree that inline logic is inevitable to apply the style based on the theme.

Reply:
I agree that's true (for now) in `ContainerHistoryLog.tsx`, where we need the `item.newState` bit to determine the coloring (which eventually it would be nice if we can figure out how to avoid; but that's a separate discussion). Here, where things just depend on the theme name, I think we can make that determination in the `makeStyles()` call, right? The style won't change while the component is rendered or be different for some of the places where it's applied. And if the theme changes, it can only happen while we're outside this view, so next time the user comes back the new theme will be in the context and `makeStyles()` would be able to create fixed styles based on that.